virtualmachine/vsphere/vm.go

```diff
@@ -735,7 +741,16 @@ func getNicInfo(vmMo mo.VirtualMachine) []VirtualEthernetCard {
 			nwcard := c.GetVirtualEthernetCard()
 			nic.MacAddress = nwcard.MacAddress
 			nic.NicName = nwcard.DeviceInfo.GetDescription().Label
-			nic.NetworkName = nwcard.Backing.(*types.VirtualEthernetCardNetworkBackingInfo).DeviceName
```

Comment:
Kindly check if we could have similar problem in other areas (cpu, mem, disk etc.)